### ISSUE TITLE
/Library/Keychains/System.keychain doesn't exist

### ISSUE THREAD

**Issue by paianoa** [darlinghq/darling#950]
**Tool**
The command `security list-keychains`  returns "/Library/Keychains/System.keychain" but the corresponding file doesn't exist.

**Example Command**
`security list-keychains`

**System Information**

`Darwin phptest 19.6.0 Darwing Kernel Version 19.6.0 x86_64`

What system are you running?

Darling a on linux debian10 with kernel:

`Linux phptest 5.10.0-0.bpo.3-amd64 #1 SMP Debian 5.10.13-1~bpo10+1 (2021-02-11) x86_64 GNU/Linux`

**Comment by facekapow** [comment]
Yeah, the whole Security framework (and related tools) are currently quite non-functional.
